app/models/user.rb

```diff
@@ -9,5 +10,6 @@ class User < ApplicationRecord
 
   validates :first_name, presence: true
   validates :last_name, presence: true
+  validates :avatar_photo, presence: true
```

Comment:
Ideally a profile picture should be optional, so perhaps we should not enforce presence.

Reply:
I initially deleted it then I saw your other comment. I will leave it like this..it's too much effort for a feature we're not even showing.